### PR TITLE
Create voicebot

### PR DESCRIPTION
**Pull request by dieguischa**

### CODE REVIEW

**Review by juan-voiq — APPROVED** [review]

File: src/components/Voicebots/CreateVoicebot/CreateVoicebot.jsx [hunk 1]
```diff
@@ -1,57 +1,229 @@
...
+  const [creationErrorMessage, setCreationErrorMessage] = useState("");
+  const [createButtonDisabled, setCreateButtonDisabled] = useState(false);
 
-  const onSubmit = (values) => {
+  useEffect(() => {
+    UsersService.readAll(
+      history,
+      (response) => {
+        setUsers(response);
+        for (const user of response) {
```
Comment (DanielOlarte) [inline src/components/Voicebots/CreateVoicebot/CreateVoicebot.jsx:35]: @dieguischa We should ignore the Super Admin user (it shouldn't have a bot associated to it)

File: src/components/Voicebots/CreateVoicebot/CreateVoicebot.jsx [hunk 2]
```diff
@@ -1,57 +1,229 @@
...
+            setCreateButtonDisabled(false);
+            history.push('/voicebots');
+          } else {
+            setCreateButtonDisabled(false);
+            setCreationErrorMessage(response.data.message);
+          }
+        }
+      );
+    } catch (e) {
+      console.error(e);
```
Comment (DanielOlarte) [inline src/components/Voicebots/CreateVoicebot/CreateVoicebot.jsx:62]: @dieguischa We should display an error message to the user if this happens

File: src/components/Voicebots/CreateVoicebot/CreateVoicebot.jsx [hunk 3]
```diff
@@ -1,57 +1,229 @@
...
+            createVoicebot(values, response.data.id);
+          } else {
+            setCreateButtonDisabled(false);
+            setCreationErrorMessage(response.data.message);
+          }
         }
       );
     } catch (e) {
-      console.log(e);
+      console.error(e);
```
Comment (DanielOlarte) [inline src/components/Voicebots/CreateVoicebot/CreateVoicebot.jsx:90]: @dieguischa Same here

File: src/services/UsersService.js [hunk 4]
```diff
@@ -4,14 +4,41 @@ const config = require('../config/voiq.json');
...
+  ).then((response) => {
+    responseCallback(response);
+  }).catch((error) => {
+    console.error("ERROR");
+    console.error(error);
+    responseCallback(error.response);
+    errorsHelper.handleAxiosError(history, error);
+  });
+}
+
```
Comment (DanielOlarte) [inline src/services/UsersService.js:20]: @dieguischa Remove extra space

**Review by DanielOlarte — CHANGES_REQUESTED** [review]
Make sure that the FAQs panel is available (even if they don't have FAQs yet) so they can create or upload a file.

**Review by DanielOlarte — APPROVED** [review]

**Pull request merged**